src/pipx/paths.py

```diff
@@ -83,24 +96,40 @@ def home(self) -> Path:
         elif self._fallback_home:
             home = self._fallback_home
         else:
-            home = Path(DEFAULT_PIPX_HOME)
+            home = Path(self._default_home)
```

Comment:
This should already be a `Path` object, IIUC.

```suggestion
            home = self._default_home
```